artiq/gateware/fmcdio_vhdci_eem.py

```diff
@@ -19,8 +19,9 @@ def _get_connectors():
         for j, pair in enumerate(eem_fmc_connections[i]):
             for pn in "n", "p":
                 cc = "cc_" if j == 0 else ""
```

Comment:
This variable is no longer used, remove

Reply:
It is used for constructing the `eem?` connectors that the `add_std()` functions for each class in `artiq.gateware.eem` can use when requesting.

Reply:
OK sorry I missed it